src/commands/commerce/extension/unmap.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { flags, SfdxCommand } from '@salesforce/command';
+import { Messages, Org, SfdxError } from '@salesforce/core';
+import { QueryResult } from '@mshanemc/plugin-helpers/dist/typeDefs';
+import { forceDataSoql, forceDataRecordDelete } from '../../../lib/utils/sfdx/forceDataSoql';
+import { StatusFileManager } from '../../../lib/utils/statusFileManager';
+import { Result } from '../../../lib/utils/jsonUtils';
+
+Messages.importMessagesDirectory(__dirname);
+
+const TOPIC = 'extension';
+const CMD = `commerce:${TOPIC}:unmap`;
+const msgs = Messages.loadMessages('@salesforce/commerce', 'store');
+
+export class UnMapExtension extends SfdxCommand {
+    public static readonly requiresUsername = true;
+    public static description = msgs.getMessage('extension.unmap.cmdDescription');
+    public static example = [
+        `sfdx ${CMD} --registered-extension-name test-extension-name --store-name test-store-name `,
+        `sfdx ${CMD} --registered-extension-name test-extension-name --store-id test-store-id `,
+    ];
+    public static flagsConfig = {
+        'registered-extension-name': flags.string({
+            char: 'r',
+            description: msgs.getMessage('extension.unmap.regExtensionNameFlagDescription'),
```

Comment:
I think it got lost in the last PR but should we make a "flag" section which stores all the flag descriptions so that the same description can be reused in each command

Reply:
I would have to modify all the existing commands' flag descriptions manually if I created a new 'flag' section. Would that be necessary?

Reply:
no problem it would be nice but not necessary since everything seems to be working